src/Websocket.Client/WebsocketClient.cs

```diff
@@ -40,21 +40,48 @@ public class WebsocketClient : IWebsocketClient
 
         private readonly BlockingCollection<string> _messagesTextToSendQueue = new BlockingCollection<string>();
         private readonly BlockingCollection<byte[]> _messagesBinaryToSendQueue = new BlockingCollection<byte[]>();
-
-
+        
         /// <summary>
         /// A simple websocket client with built-in reconnection and error handling
         /// </summary>
         /// <param name="url">Target websocket url (wss://)</param>
         /// <param name="clientFactory">Optional factory for native ClientWebSocket, use it whenever you need some custom features (proxy, settings, etc)</param>
         public WebsocketClient(Uri url, Func<ClientWebSocket> clientFactory = null)
+            : this(url, GetClientFactory(clientFactory))
+        {
+
+        }
+
+        private static Func<Uri, CancellationToken, Task<WebSocket>> GetClientFactory(Func<ClientWebSocket> clientFactory)
```

Comment:
Move it below public methods please, to follow standard .NET code guidelines: https://blog.mkotas.cz/dotnet/2016/06/04/style-guidelines.html

Reply:
Fixed